docs/partners/marketplace.md

```diff
@@ -0,0 +1,49 @@
+---
+id: marketplace 
+title: Marketplace 
+---
+
+With the release of Cortex XSOAR v.6 in August 2020, we are pleased to announce the industry's first fully integrated marketplace for orchestration content!
+
+Cortex XSOAR Marketplace is a digital storefront for discovering, exchanging and contributing security orchestration content centrally within the Cortex XSOAR platform. It enables users to install turnkey content packs to help solve any security use case with content contributed by SecOps experts and the world’s largest SOAR partner community, including you.
+
+Additionally, Partners have the ability to monetize their content on the Marketplace. You decide the content, the price, and how to best market it, and get a healthy share of the revenue. The customer gets premium content to solve their use case. It's a win-win.
+
+## Program Details
+
+This page is meant as an overview to Marketplace. Please see the [Palo Alto Networks Cortex XSOAR Partner Program Guide](/program-guide) for details on the program. 
+
+
+## Packs
+
+All content on Marketplace is included in a Pack. (see docs on packs). When Partners create a pack, you specify the content that is included in that pack. 
+
+Packs can be free or paid. Our platform does full dependency checking, too, so a pack can be dependent on, say, a command in another pack. This allows you to build a playbook addressing a full use case across products. 
+
+If you previously had a Demisto or Cortex XSOAR integration, it has been converted to a pack format and added to Marketplace for you.
+
+### Pricing
+
+All integrations must be free. We always want to encourage our mutual customers to connect our systems such that 1+1=3. Premium paid content can consist of any other elements including complex playbooks, detailed automation scripts, dashboards that appeal to a specific user type, etc. 
+
+Premium content is held in your private repository. Free content is held in our open source repository.  
+
+### Points
+
+The Marketplace transacts in points, rather than dollars, such that one point is valued at US$100. Customers purchase points via their Palo Alto Networks sales channels and spend those points on your content. 
```

Comment:
Do you want to mention that this is monthly fee?

Reply:
Good idea, will ad, thx.